biomage/account/account.py

```diff
@@ -137,19 +140,60 @@ def _create_user(full_name, email, password, userpool, overwrite=False):
     email = email.lower()
 
     error = create_account(full_name, email, userpool)
-    # if the user already exists, just proceed and change the password
-    # this way, when there's an error creating a list you can just
-    # re-run the whole script and get the correct tmp passwords
+    # Return error if we are not overwriting user password
```

Comment:
this comment is not correct, we are returning if there's an error and EITHER the error does not contain `usernameExistsException` OR we are not overwriting

Reply:
`("usernameExistsException" not in str(error) or not overwrite)` is equal to `not ("usernameExistsException" in str(error) and overwrite)`, which is what is written in the comment. I'll just use that for the equality, I think that would be much easier to understand right?

Reply:
I left it out because `"usernameExistsException" in str(error)` is a prerequisite for `overwrite` to have an effect, so I thought it would be kind of useless to put it in the comment. I think I'll just rewrite the if statement to 

    if error and not ("usernameExistsException" in str(error) and overwrite)

and delete the comment